### ISSUE TITLE
No detailed log output after run go test 

### ISSUE THREAD

**Issue by robertzhangwenjie** [smartystreets/goconvey#667]
my testcase:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/35749198/188298832-1b9180c2-1a2e-4e52-b736-7a88d264b51c.png">

after  run `go test ./... -v`, the output as below:
<img width="517" alt="image" src="https://user-images.githubusercontent.com/35749198/188298792-ac735015-c171-4e2e-bbeb-762a81ad44fd.png">

expected:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/35749198/188298821-e0cf3039-4b1e-497c-9ee9-d08b1722062a.png">

**Comment by robertzhangwenjie** [comment]
I found it works on linux os ,but doesnot work on macos with m1

**Comment by robertzhangwenjie** [comment]
I found the reason why the reuslt was not my expect, cause i used github.com/glycerine/goconvey instead goconvey in my project.

**Comment by cookchen233** [comment]
Yes, this is a bug. I have debugged it for a long time, if using "go test -v", it works, but when running the compiled test file, it does not work, and I already add "-test.v" when compiling.
go test -c -test.v
./test
./test -test.v
all of them don't have the details.

**Comment by cookchen233** [comment]
I found a parameter "-convey-story", adding it will show the details. 
./test -convey-story
It works!
